### DISCUSSION TITLE
Feature to disable Annotation Edits

### DISCUSSION THREAD

**Discussion by ibanksy** [sumatrapdfreader/sumatrapdf#2585] [General]
Hi There, 

I thought it might be a good idea to have a feature allowing you to disable editing the annotations. Since a recent update, allowing me to click an move annotations it can get quite annoying when panning up a page and accidentally clicking on or moving an annotation. Which on occasion actually crashes the pdf i'm looking at. I find myself constantly closing sumatra and reopening after each of these occasions.

Cheers
Iain

**Comment by GitHubRulesOK** [comment]
The issue is covered in https://github.com/sumatrapdfreader/sumatrapdf/issues/2154 and ***more relevant https://github.com/sumatrapdfreader/sumatrapdf/issues/1999*** and in part related https://github.com/sumatrapdfreader/sumatrapdf/issues/1995 https://github.com/sumatrapdfreader/sumatrapdf/issues/1681

However Ensure you are using latest [pre-release ](https://www.sumatrapdfreader.org/prerelease) and if that crashes on annotation please raise a fresh issue with the steps to recreate fail

**Reply by ibanksy** [reply]
Hi thanks for your reply. Ideally there'd be an option to disable moving / editing existing annotations in a pdf. I don't think that has been addressed has it? I've just installed the latest pre-release and don't see any options to do so

**Reply by GitHubRulesOK** [reply]
no options hence those related open issue's, Perhaps add a Canny suggestion https://sumatrapdf.canny.io/feature-requests

**Comment by charygao** [comment]
same require +1 , add the disable Annotation functions command settings please.THANKS.
